docs/tech-reference/iso-rsync-distributor.rst

```diff
@@ -57,8 +57,8 @@ The ``distributor_config`` contains a ``remote`` section with the following sett
 
 ``root``
   The absolute path to the remote root directory where all the data (content and published content)
-  lives. This is the remote equivalent to ``/var/lib/pulp``. The repo id is appended to the
-  ``root`` path to determine the location of published repository.
+  lives. This is the remote equivalent to ``/var/lib/pulp``. The `relative_url` of the
```

Comment:
FYI, this is the wrong way to demarcate inline code:

    my `inline` code

This is the right way to demarcate inline code:

    my ``inline`` code